go.mod

```diff
@@ -2,4 +2,7 @@ module tinygo.org/x/drivers
 
 go 1.14
 
-require github.com/eclipse/paho.mqtt.golang v1.2.0
+require (
+	github.com/eclipse/paho.mqtt.golang v1.2.0
+	github.com/frankban/quicktest v1.10.0
```

Comment:
It adds a dependency for testing, but I can't decide if it's OK or not.
What do you think? @deadprogram @aykevl

Reply:
I also wasn't sure if that would be OK either. I could just remove the test file and then it would be the same as all the other drivers, I guess :grin:

Reply:
I'm personally fine with extra dependencies, although I haven't looked into this particular one.